packages/commons/src/file-manager/fileManager.ts

```diff
@@ -12,9 +13,11 @@ import { Logger, LoggerConfig } from "../index.js";
 import {
   fileManagerCopyError,
   fileManagerDeleteError,
+  fileManagerGetError,
   fileManagerListFilesError,
   fileManagerStoreBytesError,
 } from "./fileManagerErrors.js";
+import { Readable } from "node:stream";
```

Comment:
[Github Action lint suggests](https://github.com/pagopa/interop-be-monorepo/actions/runs/9777770615/job/26993147112?pr=663#step:5:207) to change the import order importing  node:stream before 🤔